flux/src/notifications/NotificationList.tsx

```diff
@@ -34,7 +40,11 @@ export function providerNotificationClass() {
 
 export function receiverNotificationClass() {
   return makeCustomResourceClass({
-    apiInfo: [{ group: notificationGroup, version: notificationVersion }],
+    apiInfo: [
+      { group: notificationGroup, version: notificationVersion },
+      { group: notificationGroup, version: 'v1beta2' },
+      { group: notificationGroup, version: 'v1' },
```

Comment:
do we have v1beta2 for notification??
https://fluxcd.io/flux/components/notification/api/